### PR TITLE
Added fts search to notes

### PR DESCRIPTION
**Pull request by stephanheilner**

### CODE REVIEW

File: LDSAnnotationsTests/AnnotationStoreTests.swift [hunk 1]
```diff
@@ -33,13 +33,26 @@ class AnnotationStoreTests: XCTestCase {
...
         
         let annotation2 = try! annotationStore.addAnnotation(docID: docID, docVersion: 1, appSource: "Test", device: "iphone", source: .local)
-        _ = try! annotationStore.addNote(title: "Book of Mormon Stories", content: "That my teacher tells to me", annotationID: annotation2.id)
+        let note2 = try! annotationStore.addNote(title: "Book of Mormon Stories", content: "That my teacher tells to me", annotationID: annotation2.id)
         
-        let results = annotationStore.searchNotes(searchString: "teacher")
-        XCTAssertFalse(results.isEmpty)
+        // Search title
+        let results1 = annotationStore.searchNotes(searchString: "Child")
+        XCTAssertTrue(results1.count == 1)
```
Comment (nickmshelley) [inline LDSAnnotationsTests/AnnotationStoreTests.swift]: `XCTAssertEquals` gives better logs when it fails. (And on the three below as well).

**Comment by nickmshelley** [comment]
One more minor thing.

**Comment by nickmshelley** [comment]
👍

**Comment by nickmshelley** [comment]
👍

**Comment by hiltonc** [comment]
I'm late to the party, but... COOL!

**Pull request merged**